### ISSUE TITLE
Memory leak solution

### ISSUE THREAD

**Issue by ilansas** [badfortrains/wsExample#5]
Hi, 

I arrived here by looking at the issue on the react-native repo. Joshbedo gave a solution to the memory leak and I was wondering (as you didn't answered) if this was a good one and what you think about it. I just implemented it but didn't had time to run some tests. 

FYI this was his answer : https://github.com/facebook/react-native/issues/619#issuecomment-94878369

Anyway thanks for your feedback ;)

**Comment by badfortrains** [comment]
Hi Ilansas,

I should probably update the readme, I wrote it when there was no cleanup code in the websocket pollyfill so closed websockets would leak memory.  Since then I've fixed the issue, so I'm not currently aware of any memory leaks in the polyfill.

The facebook/react-native#619 (comment) is addressing a leak in the actual example that uses the polyfill.  It looks like he is using a list view to display the messages instead of simply appending a new text view for each message, which should reduce the amount of memory needed.  The point of the example is really just to show that polyfill works though, the react-native view code is naively simple and should definitely be changed in a real project.
